packages/executor/Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN npm install
 
 COPY . .
 
-CMD [ "yarn", "start" ]
+CMD [ "yarn", "start", "--network", "TODO"]
```

Comment:
We need to replace "TODO" with a the `NETWORK_NAME` env variable. Do you know how to do this? @RPate97

Reply:
So I thought, this could be acheived just by setting the `HARDHAT_NETWORK` environment variable as described in [this stackoverflow post](https://ethereum.stackexchange.com/questions/83656/where-does-the-line-blur-between-a-task-and-a-script-in-hardhat/93433#93433). I was thinking this would allow us to just specify the `HARDHAT_NETWORK` in the .env file passed to docker run. 

If using `HARDHAT_NETWORK` doesn't work, we can use the [--build-arg](https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg) option. Be aware that --build-arg is passed into the docker build command and the output container will be configured with the input argument. This means we'll need to build and maintain separate images for each network we're supporting which isn't ideal IMO.

Reply:
Ah didn't know we could do that with the env variable! Confirmed that we can verify contracts with it. Changed everything back to the way it was.